### ISSUE TITLE
Crash report alpha-64: NPE in EditPostSettingsFragment.updatePostFormatKeysAndNames

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#6319]
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.EditPostActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'int org.wordpress.android.fluxc.model.SiteModel.getId()' on a null object reference
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2665)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'int org.wordpress.android.fluxc.model.SiteModel.getId()' on a null object reference
       at org.wordpress.android.fluxc.persistence.SiteSqlUtils.getPostFormats(SiteSqlUtils.java:240)
       at org.wordpress.android.fluxc.store.SiteStore.getPostFormats(SiteStore.java:656)
       at org.wordpress.android.ui.posts.EditPostSettingsFragment.updatePostFormatKeysAndNames(EditPostSettingsFragment.java:777)
       at org.wordpress.android.ui.posts.EditPostSettingsFragment.onCreate(EditPostSettingsFragment.java:147)
       at android.app.Fragment.performCreate(Fragment.java:2336)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:949)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1171)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1153)
       at android.app.FragmentManagerImpl.dispatchCreate(FragmentManager.java:2034)
       at android.app.FragmentController.dispatchCreate(FragmentController.java:165)
       at android.app.Activity.onCreate(Activity.java:960)
       at android.support.v4.app.BaseFragmentActivityGingerbread.onCreate(BaseFragmentActivityGingerbread.java:54)
       at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:319)
       at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:85)
       at org.wordpress.android.ui.posts.EditPostActivity.onCreate(EditPostActivity.java:240)
       at android.app.Activity.performCreate(Activity.java:6679)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2618)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

**Comment by maxme** [comment]
Looks like it's happening in alpha-65

**Comment by tonyr59h** [comment]
Closed via #6351. 🤞
